guide/how-it-works/silent-payments.md

```diff
@@ -0,0 +1,360 @@
+---
+layout: guide
+title: Silent payments
+description: How silent payments impact and improve bitcoin UX
+nav_order: 11
+parent: How it works
+permalink: /guide/how-it-works/silent-payments/
+redirect_from:
+ - /guide/onboarding/creating-a-new-wallet/
+ - /guide/onboarding/first-use/
+main_classes: -no-top-padding
+image: https://bitcoin.design/assets/images/guide/how-it-works/silent-payments/silent-payments-header.png
```

Comment:
This should be a custom image that is 1200x630.

Reply:
Fixed in https://github.com/BitcoinDesign/Guide/pull/1109/commits/1617122d661dee455ee5d2e10d5e379472f540ca